### DISCUSSION TITLE
Add a token to a folder name to force recognition as a collection

### DISCUSSION THREAD

**Discussion by efalsken** [ZeroQI/Absolute-Series-Scanner#499] [Ideas]
Right now:
Sometimes a folder name is recognized as a valid series title and then automatically is recognized and ends up mis-categorizing child folders even if the child folders have valid `{anidb}` codes. For many series, if the folder name is in english, then it won't auto-recognize. But sometimes it does.

Desired:
I'd like to add `{collection}` at the end of my folder name to force the scanner to recognize the folder as a collection without the `{collection}` bit in it.

```
Gundam {collection}/MS Gundam 0079 (1979-80) [anidb-715]
Gundam {collection}/MS ZETA Gundam (1985-86) [anidb-718]
Gundam {collection}/SEED (2002-03) [anidb3-252]
Gundam {collection}/SEED DESTINY - (2004-05) [anidb3-2186]
Gundam {collection}/MSG 00 - Season 01 (2007-2009) [anidb3-5252]
Gundam {collection}/MSG 00 - Season 02 (2008) [anidb3-5903-s1]
Gundam {collection}/Gundam Build Divers (2018) [anidb-13774]
Gundam {collection}/MSG Thunderbolt - Season 01 (2015) [anidb3-11740]
Gundam {collection}/MSG Thunderbolt - Season 02 (2017) [anidb3-12549]
```

Should recognize as 6 different "Shows" but should all be in the "Gundam" collection.

**Comment by ZeroQI** [comment]
You can use grouping folders as collection field: local in agent settings for collection field

I need series specific scanner logs for the scanner importing errors

**Comment by efalsken** [comment]
Right. I'll try to reproduce it again. It's only *sometimes* and seems to be only when the parent folder is recognized as a valid series title on the first try.

**Comment by ZeroQI** [comment]
Grouping folders are a hack. While on the root folder scan, the scanner need to add subfolders as series. If it it done during the normal scanner call to the subfolder, it will put all series in a single entry linked to the folder at root so the source code disregard these calls for grouping folders

If you put videos in the grouping folders it will crap itself
